internal/cobraext/const.go

```diff
@@ -36,8 +36,8 @@ const (
 	ReportOutputFlagName        = "report-output"
 	ReportOutputFlagDescription = "output location for test report"
 
-	VerboseFlagName        = "verbose"
-	VerboseFlagDescription = "verbose mode"
+	SkipPullRequestFlagName        = "skip-pull-request"
+	SkipPullRequestFlagDescription = "skip opening a new pull request"
```

Comment:
Where are these two used? I think maybe these are not supposed to be in this PR?

Reply:
You're right, didn't clean it properly.